JPS_VIRTUALSENSOR/JurongIslandInputAgent/src/main/java/com/cmclinnovations/jurongisland/QueryClient.java

```diff
@@ -0,0 +1,155 @@
+import org.eclipse.rdf4j.sparqlbuilder.rdf.Iri;
+import org.json.JSONArray;
+
+import uk.ac.cam.cares.jps.base.derivation.Derivation;
+import uk.ac.cam.cares.jps.base.derivation.DerivationClient;
+import uk.ac.cam.cares.jps.base.derivation.DerivationSparql;
+import uk.ac.cam.cares.jps.base.interfaces.StoreClientInterface;
+import uk.ac.cam.cares.jps.base.query.RemoteRDBStoreClient;
+import uk.ac.cam.cares.jps.base.timeseries.TimeSeries;
+import uk.ac.cam.cares.jps.base.timeseries.TimeSeriesClient;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import static org.eclipse.rdf4j.sparqlbuilder.rdf.Rdf.iri;
+import org.eclipse.rdf4j.sparqlbuilder.rdf.Rdf;
+import org.eclipse.rdf4j.sparqlbuilder.rdf.RdfLiteral;
+import it.unibz.inf.ontop.model.vocabulary.GEO;
+import org.eclipse.rdf4j.model.vocabulary.RDF;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+
+import org.postgis.Point;
+
+/**
+ * Execute sparql queries to query and instantiate data
+ */
+public class QueryClient {
+    private static final Logger LOGGER = LogManager.getLogger(QueryClient.class);
+    private StoreClientInterface queryStoreClient;
+    private StoreClientInterface updateStoreClient;
+    
+
+    static final String PREFIX = "https://www.theworldavatar.com/kg/ontodispersion/";
+    public static final String CHEM = "http://theworldavatar.com/ontology/ontochemplant/OntoChemPlant.owl#" ;
+    private static final Prefix P_DISP = SparqlBuilder.prefix("disp",iri(PREFIX));
+    static final String OM_STRING = "http://www.ontology-of-units-of-measure.org/resource/om-2/";
+    private static final String ONTO_BUILD = "https://www.theworldavatar.com/kg/ontobuiltenv/" ;
+    private static final Prefix P_OM = SparqlBuilder.prefix("om",iri(OM_STRING));
+    private static final String OWL_PREFIX = "http://www.w3.org/2002/07/owl#";
+    private static final Prefix P_OWL = SparqlBuilder.prefix("owl",iri(OWL_PREFIX));
+    private static final Prefix P_CHEM = SparqlBuilder.prefix("chem",iri(CHEM));
+    private static final Prefix P_GEO = SparqlBuilder.prefix("geo", iri(GEO.PREFIX));
+    private static final Prefix P_BUILD = SparqlBuilder.prefix("build",iri(ONTO_BUILD));
+
+    // classes
+    // Since there is only CO2 emissions data for Jurong Island, other pollutants like NO2 and PM are 
+    // not listed here. 
+    private static final Iri StaticPointSource = P_DISP.iri("StaticPointSource");
+    private static final Iri CO2 = P_DISP.iri("CO2");
+    private static final Iri Density = P_OM.iri("Density");
+    private static final Iri MassFlow = P_OM.iri("MassFlow");
+    private static final Iri Temperature = P_OM.iri("Temperature");
```

Comment:
use screaming snake case for constants

Reply:
The case of those constants has been updated